BasicAlgorithms/BubbleSort.cs

```diff
@@ -5,20 +5,34 @@ namespace BasicAlgorithms
 {
     public class BubbleSort
     {
-        public static void Sort(int[] Items)
+        public static void Sort(int[] Items, bool userPref = true)
```

Comment:
userPref is not a sugestive name.  Maybe something like bool ascending=false is a more suitable option